docs/specification.md

```diff
@@ -1708,3 +1890,87 @@ Security is a paramount concern in A2A. Key considerations include:
 - **Data Privacy:** Adhere to all applicable privacy regulations for data exchanged in `Message` and `Artifact` parts. Minimize sensitive data transfer.
 
 For a comprehensive discussion, refer to the [Enterprise-Ready Features guide](./topics/enterprise-ready.md).
+
+## 11. A2A Compliance Requirements
+
+This section defines the normative requirements for A2A-compliant implementations.
+
+### 11.1. Agent Compliance
+
+For an agent to be considered **A2A-compliant**, it **MUST**:
+
+#### 11.1.1. Mandatory Transport Support
+
+- **Support JSON-RPC 2.0 transport**: All agents **MUST** implement JSON-RPC 2.0 over HTTP(S) as defined in [Section 3.2.1](#321-json-rpc-20-transport-mandatory).
+- **Expose Agent Card**: **MUST** provide a valid `AgentCard` document as defined in [Section 5](#5-agent-discovery-the-agent-card).
+- **Declare transport capabilities**: **MUST** accurately declare all supported transports in the `AgentCard` using `preferredTransport` and `additionalInterfaces` fields following the requirements in [Section 5.6](#56-transport-declaration-and-url-relationships).
+
+#### 11.1.2. Core Method Implementation
+
+**MUST** implement all of the following core methods via JSON-RPC 2.0 transport:
+
+- `message/send` - Send messages and initiate tasks
+- `tasks/get` - Retrieve task status and results
```

Comment:
Out of curiosity, if an agent never intends to create tasks, and only wants to return messages, should it not be possible for them to not implement `tasks/get` and `tasks/cancel`  ?

Reply:
Personal opinion here: yes. If using or not tasks are not configurable as a capability, I think the agent should expose the all methods